lib/model-builder.js

```diff
@@ -887,7 +892,7 @@ ModelBuilder.prototype.buildModels = function(schemas, createModel) {
  * @param {String} name The model name
  * @param {Object} json The JSON object
  * @param {Object} options The options
- * @returns {}
+ * @returns {ModelClass} The generated model definition object.
```

Comment:
nitpick: I thought model definition object is part of the constructor but not a same concept?